CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # CHANGELOG
 
+# Version 3.2.0-BETA.3
+* Add Spring integration (preview).
+* Add thread name to log capture. Resolves [#1699](https://github.com/microsoft/ApplicationInsights-Java/issues/1699)
+* Use azure monitor exporter instead of Open Telemetry exporter.
+* Add data truncation and sanitization to telemetry data.
```

Comment:
I don't think there's any end-user facing changes here, in which case no need to mention
```suggestion
```